nion/swift/ExportDialog.py

```diff
@@ -366,6 +380,7 @@ def units(self, new_units: int) -> None:
         new_enum = UnitType(new_units)
         if self.__units != new_enum:
             self.__units = new_enum
+            self.__units_model.value = self.unit_to_str_map[self.__units]
```

Comment:
The `self.__units` could just be eliminated. The value can be computed on the fly using `UnitType(self.__units_model.value or 0)` and set directly. Watch out for None values when making that change, though. You could add a new property named `_units` for use within this class.

I think as part of that change, there might need to be another property `units_index` and that new property should be the one that is used for binding in `ExportSVGHandler`. It's confusing as it is now to have `units` property referring to an `int` and `self.__units` referring to an enum.

There are many ways to handle this, but it needs to be cleaned up and clear to someone reading the code.